pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -0,0 +1,49 @@
+{stdenv, fetchurl, unzip, libX11, libcxxabi, glib, xorg, qt4, fontconfig, phonon, freetype, zlib, libpng12, libICE, libXrender, cups, lib}:
+
+stdenv.mkDerivation {
+  name = "wpsoffice-${version}";
+  version = "10.1.0.5503"
+
+  src = fetchurl {
+    name = "${name}.tar.gz";
+    url = "http://kdl.cc.ksosoft.com/wps-community/download/a20/wps-office_10.1.0.5503~a20p2_x86_64.tar.xz";
+    sha256 = "0h9f8s7zkpd056ibrj978mr04imv631sp1wljplh99l5ncns6hws";
+  };
+
+  meta = {
+    description = "Office program originally named Kingsoft Office";
+    homepage = http://wps-community.org/;
+    license = lib.licenses.unfree-redistributable;
```

Comment:
can you restrict the `meta.platforms = [ "x86_64-linux" ];`. Are there other binary builds available ?